BUILDING.md

```diff
@@ -45,6 +45,19 @@ Check that `cargo` is correctly installed by running
 cargo --version
 ```
 
+Next, install the nightly version of rustfmt
+```
+rustup toolchain install nightly
+rustup component add rustfmt --toolchain nightly
```

Comment:
It needs to be a specific toolchain that changes every release... eg nightly-2019-05-10

Reply:
ah...